src/resources/scripts/compute_embedding.py

```diff
@@ -21,11 +21,12 @@
 
 image = Image.open(in_path)
 
-if image.mode == 'RGBA':
+if image.mode == 'RGBA' or image.mode == 'L' or image.mode == 'P':
     image = image.convert('RGB')
 
-if image.mode == 'L':
-    image = image.convert('RGB')
+if image.mode =='I':
+    #I images need to be rescaled manually before converting
+    image = Image.fromarray(((np.array(img)/(2**16))*2**8).astype(np.uint8)).convert('RGB')
```

Comment:
Isn't `(img/2**16)*2**8 == img/2**8`? Or do you want to leave it like that because it explains better what's going on?

Reply:
I don't really mind, but it might be easier to read. /2**8 might suggest that we are scaling to [0,1] as this is mostly done by /256. Probably we could add in the comment that I is a 32bit integer image.